[1,"Error occurred during cleanup: Error: El proceso no tiene acceso al archivo porque est√° siendo utilizado por otro proceso. (os error 32): remove 'C:\\Users\\sergi\\AppData\\Local\\Temp\\quarto-session981484e8ffd5cd6b'\n"]
